src/compiler/types.ts

```diff
@@ -3798,7 +3809,9 @@ namespace ts {
         ContainsYield = 1 << 24,
         ContainsHoistedDeclarationOrCompletion = 1 << 25,
 
-        HasComputedFlags = 1 << 29, // Transform flags have been computed.

```

Comment:
It won't be...This is an internal flag used by emitter. No one should take dependency on exact value of this flag